bench/bench-local.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh -e
+
+mkdir -p $1
+WRKDIR=`realpath $1`/bench
+scriptdir=$(CDPATH='' cd -- "$(dirname -- "$0")" && pwd -P)
+
+
+start_kvlds () {
+	mkdir $WRKDIR
+	${scriptdir}/../lbs/lbs -d $WRKDIR -s $WRKDIR/sock_lbs -b 2048 -p $WRKDIR/lbs.pid
+	${scriptdir}/../kvlds/kvlds -s $WRKDIR/sock_kvlds -l $WRKDIR/sock_lbs -p $WRKDIR/kvlds.pid
+}
+stop_kvlds () {
+	kill `cat $WRKDIR/kvlds.pid`
+	kill `cat $WRKDIR/lbs.pid`
+	rm -rf $WRKDIR
+}
+
+if [ $# -ne 1 ]; then
+	echo "usage: bench-local.sh DIRECTORY"
+	exit 1
+fi
+
+# Testing bulk insert performance
+start_kvlds
+${scriptdir}/mkpairs/mkpairs `expr 1024 \* 1024 \* 130` > $WRKDIR/pairs
+${scriptdir}/bulk_insert/bulk_insert $WRKDIR/sock_kvlds < $WRKDIR/pairs |
+    lam -s 'bulk_insert ' -
```

Comment:
`lam` isn't POSIX, and isn't present on Linux.  This may or may not matter.  The freeBSD man page says "Some of the functionality of lam is standardized as the paste(1) utility by IEEE Std 1003.2 (“POSIX.2”)."

Reply:
How would you feel about `awk '{print "bulk_insert: " $0}` ?

Reply:
That would be fine.